.github/workflows/backend.yml

```diff
@@ -107,6 +107,51 @@ jobs:
       - name: Handle artifacts
         uses: ./.github/actions/artifacts
 
+  backend-test-in-silo-modes:
+    if:
```

Comment:
Will add back the needs backend files check after validating the secondary element here works.